Tests/Bricks/ImageBrickTests.swift

```diff
@@ -51,6 +51,11 @@ class ImageBrickTests: XCTestCase {
         imageURL = NSURL(fileURLWithPath: NSBundle(forClass: self.classForCoder).pathForResource("image0", ofType: "png")!)
     }
 
+    override func tearDown() {
+        super.tearDown()
+        BrickCollectionView.imageDownloader = NSURLSessionImageDownloader() // Reset the image downloader back to the default one (if a test had overwritten this)
```

Comment:
Do all brick collection views share this image downloader? Could we move injecting specific downloader instances into individual brick collection view instances, or would that pose a problem?